### ISSUE TITLE
Endless loading on administration panel

### ISSUE THREAD

**Issue by Molx32** [gophish/gophish#2174]
**Gophish version**: 0.11.0 
**Hosted on**: Azure VM - Ubuntu 18.04 LTS
**Web browser used**: Opera, Chrome (both latest versions at the time of this post)

**Brief description of the issue**: 
I followed the [documentation](https://docs.getgophish.com/user-guide/installation#running-gophish) to install Gophish using the `go build` command. When I access to the admin panel, I noticed two issues :

1. The _[Email template](https://mydomain.com:3333/templates)_ page shows an endless loading icon. The console shows that two files could not be retrieved from the web server (_c.f._ first screenshot). We can still click on the  _New template_ button, which opens a form. Once filled, clicking on the _Save template_ button does not send any data and generates a console error (_c.f._ second screenshot).
2. I observed the same behavior when accessing the _Landing pages_ page.

<img width="960" alt="Erreur_1" src="https://user-images.githubusercontent.com/16539754/115624983-b27d9e00-a2fb-11eb-9787-dc11eceef4d8.png">

<img width="958" alt="Erreur_2" src="https://user-images.githubusercontent.com/16539754/115624379-d7254600-a2fa-11eb-9eb9-80ce0184b827.png">

Since I don't know anything about Go or how the web server runs, I simply used the following command to find any file or folder containing the _vendor_ string in the gophish root folder. This command does not match any file.
`ls -R | grep -i ckeditor`

**Reproduction steps**

1. Install Gophish
2. Run Gophish with default config (or with the admin panel facing Internet)
3. Authenticate as `admin` and change password
4. Access broken pages by clicking on the lateral menu

**Comment by grovervas** [comment]
I have the same problem, the manual was followed step by step, but both the email template page and the landing page have endless load, and you cannot move forward.

**Comment by glennzw** [comment]
I've struggled to reproduce this one. The error in the console is indicating that the CKEDITOR code cant' load (that's the 3rd party editor we use to edit templates). 

Your browser is complaining that two files don't exist: `ckeditor.js` and `jquery.js`. Could you try manually click those URLs in the console to see if the pages load? It almost seems like it's a network issue and your browser couldn't load those files. 

Next up, check your gophish directory - you should have the file there:

```bash
➜  gophish $ file static/js/src/vendor/ckeditor/ckeditor.js 
static/js/src/vendor/ckeditor/ckeditor.js: UTF-8 Unicode (with BOM) text, with very long lines
```

**Comment by Molx32** [comment]
I can confirm that the **vendor** folder is not available in **static/js/src/**. The only folder here is **app/**.

**Comment by glennzw** [comment]
Odd. Did you download a prebuilt or compile from source?

Could you try install again; perhaps something interrupted the process and some files were left behind.

You should be able to access https://127.0.0.1:3333/js/src/vendor/ckeditor/ckeditor.js

**Comment by trevormac789** [comment]
Same for me /static//js/src only has the app directory. Vendor is missing.

Email Templates and Landing just spin forever like others have said. 

I'd rather not rebuild this server. Will go build leave all my configs intact and pull missing files? Be easier if I could just download the missing directory and copy it in place.

**Comment by trevormac789** [comment]
I tried **go build** again but it didn't do anything.

I downloaded the sourcecode from github, extracted and uploaded the "vendor" directory into the correct location and changed ownership, permissions and restart gophish. Seems to be working for me now.

**Comment by ifoam** [comment]
I am also experiencing this issue on a new install. https://127.0.0.1:3333/js/src/vendor/ckeditor/ckeditor.js doesn't work for me and I cannot find the file on the system.  I used the information from: https://golang.org/doc/install to install.

**Comment by glennzw** [comment]
Closing out as haven't heard back. Feel free to re-open.

**Comment by c4chale** [comment]
I am facing the same issue.
I installed GoPhish using `go install github.com/gophish/gophish@latest`; `cd`ing into the gophish pkg folder to `go build` which gave me the error in issue #2600 yet manage to use the workaround. After logging in the admin area I tried to create the template but it goes in this endless loading loop (as shown in the images). Checking on my installation I can't see the folder `vendor` in `static/js/src/`

**Comment by glennzw** [comment]
Hi all - I managed to track down the problem - `go install` silently fails because gophish has vendor folders, which is a reserved word (see https://github.com/golang/go/issues/48332).

The solution is to install via git:

```git clone https://github.com/gophish/gophish.git``` and then `cd` into the folder and `go build` as usual.

I've updated the [README.md](https://github.com/gophish/gophish/blob/master/README.md) to reflect this.
